packages/cli/src/cli/index.ts

```diff
@@ -214,6 +214,7 @@ const main = async () => {
     .option("--js, --javascript", "Initialize as a JavaScript project.")
     .option("--forge", "Initialize as a Forge project.")
     .option("--hardhat", "Initialize as a Hardhat project.")
+    .option("--extension", "Create a smart contract extension.")
```

Comment:
not for right now, but wonder if we should make this a 3rd option from `npx create app/contract/extension`

Reply:
yes please just make it `extension` namespace